index.html

```diff
@@ -615,6 +700,61 @@ <h3>Instantiate Cryptosuite</h3>
             </ol>
           </li>
           <li>
+If |options|.|cryptosuite| is `experimental-mldsa-2024` then:
+            <ol class="algorithm">
+              <li>
+Set |cryptosuite|.|createProof| to the algorithm in Section
```

Comment:
This change is *usually* appropriate across this and all related documents. It is *rare* that the value of a variable should actually be set to an algorithm.
```suggestion
Set |cryptosuite|.|createProof| to the result of running the algorithm in Section
```